great_expectations/datasource/fluent/file_path_data_asset.py

```diff
@@ -59,7 +61,12 @@ class _FilePathDataAsset(DataAsset):
     }
 
     # General file-path DataAsset pertaining attributes.
-    batching_regex: Pattern
+    batching_regex: Pattern = MATCH_ALL_PATTERN
+    # TODO: better name
+    dc_options: Mapping = pydantic.Field(
```

Comment:
@Kilo59 If we are going to go with this (not sure yet, but supposing that we will), then I would propose fully spelling out "data_connector" rather than "dc".  According to modern best practices published by a few large software developer houses, the recommendation is to spell literals out, rather than use acronyms (where confusion may arise).  In other words, we do not need to spell out HTTP or XML :-) -- thanks for your consideration.

Reply:
I'm happy to make this name this whatever.
On second thought though my inclination is for something like `connect_options`.
DataConnectors are meant to know be an internal implementation detail so including it in the name runs counter to that effort.